<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WateringDemandPatterns</class>
 <widget class="QDialog" name="WateringDemandPatterns">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Watering Demand Patterns</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QPushButton" name="loadDemandPatterns">
     <property name="text">
      <string>Load  Pattern</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="createPatternLavel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Select new pattern name:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QPushButton" name="removeValue">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QPushButton" name="createNewButton">
     <property name="text">
      <string>Create New Pattern</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QPushButton" name="newValue">
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLineEdit" name="newPatternLine"/>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="deletePattern">
     <property name="text">
      <string>Delete Pattern</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QCheckBox" name="createCheckBox">
     <property name="text">
      <string>Create New Pattern</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Load Existing Pattern:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="9" column="2">
    <widget class="QPushButton" name="uploadPatterns">
     <property name="text">
      <string>Upload Patterns</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QPushButton" name="getValues">
     <property name="text">
      <string>Get Values</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
